src/main/resources/view/ResultDisplay.fxml

```diff
@@ -4,13 +4,11 @@
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
-<?import javafx.scene.layout.Region?>
 
 <StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/17"
-    xmlns:fx="http://javafx.com/fxml/1">
-    <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" />
-    <VBox alignment="BOTTOM_CENTER">
-      <Button fx:id="resultVisitUG" text="Visit User Guide" />
-      <Region prefHeight="40"/>
```

Comment:
the removal of this, does it remove the bottom margin? i.e. will the button be anchored right at the bottom of the vbox? i would think a more sensible solution would be just to increase the minimum height enforcement of the main window

Reply:
e.g. https://github.com/AY2425S1-CS2103-F12-3/tp/commit/e051cc936b3443aaa224c2a2980d99b8e043738e which was pushed before the PE-D but did not make it to v1,5 due to lack of time

im ok with any solution as long as it looks good and works well